cmd/mountlib/mount.go

```diff
@@ -196,7 +224,7 @@ will see all files and folders immediately in this mode.
 	flags.FVarP(flagSet, &MaxReadAhead, "max-read-ahead", "", "The number of bytes that can be prefetched for sequential reads.")
 	flags.StringArrayVarP(flagSet, &ExtraOptions, "option", "o", []string{}, "Option for libfuse/WinFsp. Repeat if required.")
 	flags.StringArrayVarP(flagSet, &ExtraFlags, "fuse-flag", "", []string{}, "Flags or arguments to be passed direct to libfuse/WinFsp. Repeat if required.")
-	//flags.BoolVarP(flagSet, &foreground, "foreground", "", foreground, "Do not detach.")
+	flags.BoolVarP(flagSet, &Foreground, "foreground", "", Foreground, "Do not detach.")
```

Comment:
This changed the default to background... There are quite a lot of rclone mount users and changing the default might be unfriendly....  What do you think?

Reply:
I thought about it while building this and I think it's a drastic change. Default foreground is much nicer, won't change how existing users interact with mount.

Reply:
I suggest we change the name of the flag to `--background` then too!

You might want to write in the docs also that if using `--background` you'll probably want to use `--log-file filename` or `--syslog` to capture the output of the mount.

Reply:
"--daemon" like rsync? would be familiar for many

Reply:
I like `--daemon` better